modules/rhel-compute-updating.adoc

```diff
@@ -18,6 +18,17 @@ Because the RHEL machines require assets that are generated by the cluster to
 complete the update process, you must update the cluster before you update
 the RHEL compute machines in it.
 ====
+* Stop and disable firewalld on the host:
```

Comment:
@bysnupy, please move this step to after the Procedure heading and make it start with a "." instead of an "*".

Reply:
@kalexand-rh , thank you for your review. I've modified as you mentioned.